### PR TITLE
Add AuthZ example (GEA-12591)

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

File: examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+			// Create a tuple.
+			client.tupleCreate(new TupleCreateRequest.Builder(new Tuple[] { tuple }).build());
+			System.out.printf("User '%s' is a 'reader' for folder '%s'.\n", userId, folderId);
+
+			// Find the tuple that was just created.
+			var filter = new FilterTupleList();
+			filter.resourceNamespace().set("folder");
+			filter.resourceID().set(folderId);
+			var listResponse = client.tupleList(new TupleListRequest.Builder().setFilter(filter).build());
+			listResponse.getResult().getTuples();
```
Comment (pangea-andrest) [inline examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java:48]: Would be nice to print this results

File: examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+			// Check if the user is an editor of the folder.
+			var checkResponse = client.check(
+				new CheckRequest.Builder()
+					.setResource(folder)
+					.setAction("editor")
+					.setSubject(user)
+					.build()
+			);
+			checkResponse.getResult().isAllowed();
```
Comment (pangea-andrest) [inline examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java:59]: Here too, and the other ones

**Review by pangea-andrest — CHANGES_REQUESTED** [review]

File: examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java (hunk 1 above)
Comment (kenany) [reply examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java:48]: `Tuple`s don't print nicely though.

File: examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java (hunk 2 above)
Comment (kenany) [reply examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java]: Done.

File: examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java (hunk 1 above)
Comment (pangea-andrest) [reply examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java:48]: We could access response's fields and format them in an user friendly way, that way we show to users how to access those fields too

**Review by pangea-andrest — APPROVED** [review]
Minor comment

File: examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java (hunk 1 above)
Comment (kenany) [reply examples/authz/folder_user/src/main/java/cloud/pangeacyber/examples/App.java]: Done.

**Review by pangea-andrest — APPROVED** [review]

**Pull request merged**